### DISCUSSION TITLE
Generator for Storybook 8.0 @nx/storybook:migrate-8

### DISCUSSION THREAD

**Discussion by jorgeramirezamora** [nrwl/nx#27363] [Feature Requests]
Hello Nx team,

It would be great to add a Nx generator to go from Storybook 7 to Storybook 8 format. Similar to: https://nx.dev/nx-api/storybook/generators/migrate-7

References:

https://storybook.js.org/docs/migration-guide

**Comment by melutovich** [comment]
@jorgeramirezamora 
I'm not sure when it appeared on their website, but is the following what you are looking for?

https://nx.dev/nx-api/storybook/generators/migrate-8

**Reply by jorgeramirezamora** [reply]
This is exactly what I was looking for. Thanks for letting me know
